Create a guide for FAIR software publishing

jensdebruijn opened the pull request:
This is an initial draft, let's see if it builds first 😊

Jolien-S reviewed (changes requested):
Hi @jensdebruijn, thanks for uploading this guide. Please see the review for suggestions for adjustments and some questions/discussion points. The guide needs a bit more work before it can be published. We can also think about another picture. Maybe we have a nice one from the research software campaign. I'll have a look.

peer35 commented:
@jensdebruijn one suggestion: please emphasize researchers must never put secret or sensitive information in a git repository, even if it is private. Maybe at a section on using `.gitignore` and some pointers on how to put api keys, passwords and stuff in a config or .env file.

jensdebruijn commented:
@Jolien-S Thanks for the feedback! I have addressed all comments. We still need a good image (which I copied from the DMP). Who is in charge of making those?

Jolien-S commented:
Thanks for addressing my comments @jensdebruijn 🙏 I think it's nice to use an existing image from the RSM campaign; the sticker spaghetti code vs FAIR lasagna fits well with this guide, I think. I will add it to our repo and include it in the guide.

Jolien-S commented on guides/publishing-fair-software.qmd:
```diff
@@ -0,0 +1,246 @@
+---
+title: "How can you publish FAIR software"
+image: '/public/cover-images/DALLE_DMP.jpg'
```
```suggestion
image: '/public/RSM_sticker.png'
```

Jolien-S commented on guides/publishing-fair-software.qmd:
````diff
@@ -0,0 +1,246 @@
+../topics/software-licensing.qmd
+```
+
+## DOI
````
```suggestion
## Persistent Identifier
```

Jolien-S commented on guides/publishing-fair-software.qmd:
```diff
@@ -0,0 +1,246 @@
+
+## DOI
+
+A DOI, or Digital Object Identifier, string assigned to a document, dataset, or object to provide a permanent link. DOIs ensure that even if the URL of the content changes, the DOI link remains constant, making it a reliable way to cite and access digital materials. There are two ways to do this. Depending on how often you intend to update your code, one or the other may be simpler.
```
```suggestion
A [persistent identifier](../topics/persistent-identifier.qmd)  is a durable reference to a digital dataset, document, website or other object. A Digital Object Identifier (DOI) is a widely used Persistent Identifier in the research domain and hence for research software as well. There are two ways to generate a DOI for your code. Depending on how often you intend to update your code, one or the other may be simpler.
```
